nbs/animation.ipynb

```diff
@@ -41,50 +43,36 @@
         "%%capture\n",
         "#@title Connect to the Stability API\n",
         "\n",
-        "from base64 import b64encode\n",
         "import datetime\n",
         "import os\n",
-        "from pathlib import Path\n",
+        "import panel as pn\n",
+        "import param\n",
         "import shutil\n",
         "import subprocess\n",
         "import sys\n",
         "\n",
+        "from base64 import b64encode\n",
+        "from IPython import display\n",
+        "from pathlib import Path\n",
+        "from PIL import Image\n",
+        "from tqdm import tqdm\n",
+        "\n",
         "\n",
+        "# install Stabiliy SDK for Python\n",
         "path = Path('stability-sdk')\n",
-        "if path.exists():\n",
-        "    shutil.rmtree(path)\n",
-        "    !pip uninstall -y stability-sdk\n",
-        "\n",
-        "!git clone -b dmarx.anima.dev --recurse-submodules https://github.com/Stability-AI/stability-sdk\n",
-        "!touch ./stability-sdk/src/stability_sdk/interfaces/__init__.py\n",
-        "!pip install ./stability-sdk\n",
+        "if not is_colab and os.path.exists(\"../src/stability_sdk\"):    \n",
+        "    sys.path.append(\"../src\") # use local SDK src\n",
+        "else:\n",
+        "    if path.exists():\n",
+        "        shutil.rmtree(path)\n",
+        "        !pip uninstall -y stability-sdk\n",
+        "    !git clone -b anima_stable --recurse-submodules https://github.com/Stability-AI/stability-sdk\n",
```

Comment:
i wonder how hard it would be to populate the branch name here via CI? that'd make it really easy to continue using this as our main experimentation sandbox moving forward. checkout a new branch for some experiment, have the notebook rebuild after each push injecting the current branch name into the notebook

Reply:
yeah a little `sed` action would be cool. for now I was thinking of just pinning to particular commit when dropping for testers.